Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -3551,6 +3555,10 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
     WebCore::FloatBoxExtent m_obscuredContentInsets;
 #if PLATFORM(MAC)
+    // When there's an attached inspector, we compute and redistribute the client-set obscured content insets
+    // to the underlying page and the inspector. This field records the original client-set insets.
+    WebCore::FloatBoxExtent m_perceivedObscuredContentInsets;
```

Comment:
In that case, I think this should be called something like `m_obscuredContentInsetsFromClient` to clarify that this value comes directly from the client (the "perceived" in here makes it sound more like the "effective" insets perceived by the user).

(I think you can also remove this comment, if you rename this member)